mustermann/lib/mustermann/expander.rb

```diff
@@ -2,6 +2,7 @@
 require 'mustermann/ast/expander'
 require 'mustermann/caster'
 require 'mustermann'
+require 'addressable/uri'
```

Comment:
What is this?
I would not like to use the `addressable` gem. Mustermann has not been depended on external gems.
Is there a clear reason to add this gem?
If so, why do not you add this to gemspec.